internal/test/wsu/wsu_test.go

```diff
@@ -103,69 +70,25 @@ ansible_winrm_server_cert_validation=ignore`, ip, password, clusterAddress))
-func getKubeClient(kubeconfig string) (*kubernetes.Clientset, error) {
-	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
-	if err != nil {
-		return nil, fmt.Errorf("could not build config from flags: %v", err)
-	}
-	clientset, err := kubernetes.NewForConfig(config)
-	if err != nil {
-		return nil, fmt.Errorf("could not create k8s clientset: %v", err)
-	}
-	return clientset, nil
-}
-
 // TestWSU creates a Windows instance, runs the WSU, and then runs a series of tests to ensure all expected
 // behavior was achieved. The following environment variables must be set for this test to run: KUBECONFIG,
 // AWS_SHARED_CREDENTIALS_FILE, ARTIFACT_DIR, KUBE_SSH_KEY_PATH, WSU_PATH, CLUSTER_ADDR
 func TestWSU(t *testing.T) {
-	require.NotEmptyf(t, kubeconfig, "KUBECONFIG environment variable not set")
-	require.NotEmptyf(t, awsCredentials, "AWS_SHARED_CREDENTIALS_FILE environment variable not set")
-	require.NotEmptyf(t, dir, "ARTIFACT_DIR environment variable not set")
-	require.NotEmptyf(t, privateKeyPath, "KUBE_SSH_KEY_PATH environment variable not set")
-	require.NotEmptyf(t, playbookPath, "WSU_PATH environment variable not set")
-	require.NotEmptyf(t, clusterAddress, "CLUSTER_ADDR environment variable not set")
-
-	var err error
-	k8sclientset, err = getKubeClient(kubeconfig)
-	require.NoError(t, err)
-	existingWindowsNodes, err := k8sclientset.CoreV1().Nodes().List(metav1.ListOptions{LabelSelector: windowsLabel})
-	require.NoError(t, err)
-	// The test should fail, if there are existing Windows nodes in the cluster as we're just relying on the
-	// windowsLabel to be present on the node
-	// TODO: Label or annotate the windows node to have more determinism when running test suite.
-	require.Equalf(t, len(existingWindowsNodes.Items), 0, "expected 0 windows nodes to be present but found %v",
-		len(existingWindowsNodes.Items))
-
-	// TODO: Check if other cloud provider credentials are available
-	if awsCredentials == "" {
-		t.Fatal("No cloud provider credentials available")
-	}
-	err = createAWSWindowsInstance()
-	require.NoErrorf(t, err, "Error spinning up Windows VM: %s", err)
-	require.NotNil(t, createdInstanceCreds, "Instance credentials are not set")
-	defer cloud.DestroyWindowsVMs()
 	// In order to run the ansible playbook we create an inventory file:
 	// https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html
-	hostFilePath, err := createHostFile(createdInstanceCreds.GetIPAddress(), createdInstanceCreds.GetPassword())
+	hostFilePath, err := createHostFile(framework.Credentials.GetIPAddress(), framework.Credentials.GetPassword())
 	require.NoErrorf(t, err, "Could not write to host file: %s", err)
 	cmd := exec.Command("ansible-playbook", "-vvv", "-i", hostFilePath, playbookPath)
 	out, err := cmd.CombinedOutput()
 	require.NoError(t, err, "WSU playbook returned error: %s, with output: %s", err, string(out))
 
+	k8sclientset = framework.K8sclientset
```

Comment:
Shouldnt we delete the k8sclientset and just use framework.K8sclientset at this point?